### ISSUE TITLE
scigraph vocalbulary search service result ordering

### ISSUE THREAD

**Issue by tgbugs** [SciCrunch/SciGraph#100]
The vocabulary search service http://matrix.neuinfo.org:9000/scigraph/vocabulary/search/ does not return results in expected order. For example a search for dendrite only returns an exact match as the 50th result. http://matrix.neuinfo.org:9000/scigraph/vocabulary/search/dendrite?limit=100&searchSynonyms=false&searchAbbreviations=false&searchAcronyms=false

I think expected behavior would be to return exact matches first followed by nodes where category is not null, and then everything else after that.

**Comment by memartone** [comment]
Yes, that is in indeed the right order.  We’ve had that problem in SciCrunch results ranking too.
